src/vs/workbench/browser/parts/activitybar/activitybarActions.ts

```diff
@@ -200,7 +200,10 @@ export class AccountsActionViewItem extends ActivityActionViewItem {
 						return this.authenticationService.signOutOfAccount(sessionInfo.providerId, accountName);
 					});
 
-					const actions = hasEmbedderAccountSession ? [manageExtensionsAction] : [manageExtensionsAction, signOutAction];
+					const actions = [manageExtensionsAction];
+					if (!hasEmbedderAccountSession || authenticationSession?.canSignOut) {
```

Comment:
@RMacfarlane Added `canSignOut` option to enable sign out action for playground so that allowing user to can sign out from playground.

Reply:
LGTM, but I'm not sure I understand why we want to allow sign out there. Just to be able to test the full auth flow?

Reply:
Otherwise there'll be no other way to sign out from the app. Do not want to implement ui in the app instead use workbench's.